plugins/search-indexer/services/search-indexer.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+/**
+ * search-indexer.js service
+ *
+ * @description: A set of functions similar to controller's actions to avoid code duplication.
+ */
+
+module.exports = {
+
+};
```

Comment:
is this file necessary?

Reply:
This is how Strapi constructed the plugin when I created it with the Strapi CLI tool, so I wouldn't touch it.